### ISSUE TITLE
PHP Warning - Declaration of TightenCo\Jigsaw\Filesystem::allFiles($directory)

### ISSUE THREAD

**Issue by wesleyhf** [tightenco/jigsaw#52]
```sh

$ ./vendor/bin/jigsaw init
PHP Warning:  Declaration of TightenCo\Jigsaw\Filesystem::allFiles($directory) should be compatible with Illuminate\Filesystem\Filesystem::allFiles($directory, $hidden = false) in /home/wesley/code/the-burgers/vendor/tightenco/jigsaw/src/Filesystem.php on line 6
PHP Stack trace:
PHP   1. {main}() /home/wesley/code/the-burgers/vendor/tightenco/jigsaw/jigsaw:0
PHP   2. spl_autoload_call() /home/wesley/code/the-burgers/vendor/tightenco/jigsaw/jigsaw:55
PHP   3. Composer\Autoload\ClassLoader->loadClass() /home/wesley/code/the-burgers/vendor/tightenco/jigsaw/jigsaw:55
PHP   4. Composer\Autoload\includeFile() /home/wesley/code/the-burgers/vendor/composer/ClassLoader.php:301
PHP   5. include() /home/wesley/code/the-burgers/vendor/composer/ClassLoader.php:412
Site initialized successfully!

```

**Comment by wesleyhf** [comment]
This pull request solve the problem #49

**Comment by besologic** [comment]
Resolved in [v0.6.1](https://github.com/tightenco/jigsaw/releases/tag/v0.6.1) Thanks for reporting it!
